hummingbird/ml/_container.py

```diff
@@ -99,3 +100,26 @@ def predict_proba(self, *inputs):
         On classification tasks returns the probability estimates.
         """
         return self.forward(*inputs)[1].cpu().numpy()
+
+
+class PyTorchBackendModelAnomalyDetection(PyTorchBackendModel):
```

Comment:
What about extending `PyTorchBackendModelRegression` rather then `PyTorchBackendModel`? You can then add the comment `On anomaly detection (e.g. isolation forest) returns the predicted classes (-1 or 1).` directly ofter `On classification...`

Reply:
make sense, will do!